### ISSUE TITLE
Erroneous autocomplete popups

### ISSUE THREAD

**Issue by aaronkavlie-wf** [tinloaf/autocomplete-plus-python-jedi#28]
When I type a string key for a dictionary, I get a popup with builtin functions:

``` python
foo = {
    'bar': <- popup happens here
```

When I enter the value as a key, then a comma, it happens again.

This occurence is particularly intrusive, as it means that enter autocompletes to the `abs()` function rather than going to the next line.

**Comment by tinloaf** [comment]
Yeah, Jedi is completing to basically any valid function / symbol (builtin or not) at some points. I filed an issue regarding this with Jedi, but I think I'll just build a workaround. The only thing is: how to determine reliably if this now is a "here's all symbols"-autocompletion?

**Comment by norman123123** [comment]
This is huge, has it been fixed yet?
